packages/core/lib/generators/integrations/image_integration.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter_gen_core/generators/integrations/integration.dart';
+import 'package:flutter_gen_core/settings/asset_type.dart';
+
+class ImageIntegration extends Integration {
```

Comment:
Consider adding docs for the integration.

Reply:
Added